src/main/java/com/eva/model/Model.java

```diff
@@ -150,4 +162,45 @@
      * @throws NullPointerException if {@code predicate} is null.
      */
     void updateFilteredStaffList(Predicate<Staff> predicate);
+
+    /**
+     * Replaces address book data with the data in {@code addressBook}.
+     */
+    void setApplicantDatabase(ReadOnlyEvaDatabase<Applicant> personDatabase);
+
+    /** Returns the EvaDatabase */
+    ReadOnlyEvaDatabase<Applicant> getApplicantDatabase();
+
+    /**
+     * Returns true if a person with the same identity as {@code person} exists in the address book.
+     */
+    boolean hasApplicant(Applicant person);
+
+    /**
+     * Deletes the given person.
+     * The person must exist in the address book.
+     */
+    void deleteApplicant(Applicant target);
+
+    /**
+     * Adds the given person.
+     * {@code person} must not already exist in the address book.
+     */
+    void addApplicant(Applicant person);
+
+    /**
+     * Replaces the given person {@code target} with {@code editedPerson}.
+     * {@code target} must exist in the address book.
+     * The person identity of {@code editedPerson} must not be the same as another existing person in the address book.
```

Comment:
not sure if we wanna change it to editedApplicant, editedStaff. Can choose to change or not up to u